api/v1alpha1/collectors.go

```diff
@@ -0,0 +1,27 @@
+package v1alpha1
+
+// Collector represents a reference to a Collector to be executed as part of the release workflow.
+// +kubebuilder:object:generate=true
+type Collector struct {
+	// Name of the collector
+	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// +required
+	Name string `json:"name"`
+
+	// Collector is the type of collector to be used
+	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// +required
+	Collector string `json:"collector"`
```

Comment:
What do you think about naming this type? I assume this will be like `jira` for example. collector.collector seems a little weird to me

Reply:
Sure. Makes sense. I went with what we had in the ticket but it sounded weird to me.

Reply:
The json tag wasn't updated

Reply:
Right!